### PR TITLE
docs(synthetics_monitor): mentions NerdGraph is used.

### PR DESCRIPTION
**Pull request by matthewhartstonge**
Makes it explicit that NerdGraph is used and that the `AWS_` prefix should be removed.

# Description

Came across an issue where the link mentioned using a specific region, so created a copy+paste error as we didn't know NerdGraph was used under the hood until we compared the example and assumed that must be the issue.

## Type of change

Please delete options that are not relevant.

- [x] This change requires a documentation update

## Checklist:

Please delete options that are not relevant.

- [x] My commit message follows [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation

## How to test this change?

Please describe how to test your changes. Include any relevant steps in the UI, HCL file(s), commands, etc

- Copy text of documentation.
- Paste into https://registry.terraform.io/tools/doc-preview
- Verify documentation layout.

### CODE REVIEW

**Comment by mbazhlekova** [comment]
@matthewhartstonge Thank you for your contribution! Would you be open to also updating the docs for the other synthetics monitor types with the same information?

**Comment by matthewhartstonge** [comment]
No worries, will do ASAP 😄

File: website/docs/r/synthetics_monitor_step.html.markdown [hunk 1]
```diff
@@ -39,7 +39,7 @@ The following are the common arguments supported for `STEP` monitor:
 * `account_id`- (Optional) The account in which the Synthetics monitor will be created.
 * `name` - (Required) The name for the monitor.
 * `uri` - (Required) The uri the monitor runs against.
-* `locations_public` - (Required) The location the monitor will run from. Valid public locations are https://docs.newrelic.com/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips/. Exactly one of `locations_public` or `locations_private` is required.
+* `locations_public` - (Required) The location the monitor will run from. Valid public locations are https://docs.newrelic.com/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips/. You don't need the `AWS_` prefix as the provider uses NerdGraph. At least one of either `locations_public` or `location_private` is required.
```
Comment (matthewhartstonge) [inline website/docs/r/synthetics_monitor_step.html.markdown:42]: Slight wording variation here, from exactly to "At least one", which aligns itself with the other four resources. 

Is the change in wording okay?

Comment (mbazhlekova) [reply website/docs/r/synthetics_monitor_step.html.markdown:42]: That's a good catch! It should be "At least one of"

**Review by mbazhlekova — APPROVED** [review]

**Comment by kidk** [comment]
Thanks a lot for your PR @matthewhartstonge ❤️

**Pull request merged**